### ISSUE TITLE
List Index Range related to Material

### ISSUE THREAD

**Issue by enzyme69** [alessandro-zomparelli/tissue#10]
![screen shot 2017-05-09 at 23 16 53](https://cloud.githubusercontent.com/assets/3952687/25852741/bc5ae082-350d-11e7-887a-02e299ed5d3a.png)

I kept getting this error message when using the Tissue Add-On, not sure why. Probably because it was expecting material?

**Comment by schroef** [comment]
I cant replicate this, what what your workflow?

**Comment by alessandro-zomparelli** [comment]
Hi enzyme69, it should work with no materials. Are you using Tessellate, Settings or Refresh? Does it always occur?

**Comment by schroef** [comment]
PS Alessandro isnt it better to use try: for getting the list, if a error occurs it will be suppressed and the code will move on.
`        try:
            for i in range(len(component_materials)):
                bpy.ops.object.material_slot_add()
                bpy.context.object.material_slots[i].material = component_materials[i]
            for i in range(len(new_ob.data.polygons)): new_ob.data.polygons[i].material_index = polygon_materials[i]
        except: 
            pass`

**Comment by alessandro-zomparelli** [comment]
Yes, it's correct, I didn't used it because it seems to work fine in different conditions, with materials and without materials, but maybe I should add that.

**Comment by alessandro-zomparelli** [comment]
This issue should be fixed in the version 0.3.
In the new version there is no more a Tissue panel, but the operations are now in the "Create" panel and "Tools" panel.
